hipcheck/src/analysis/score.rs

```diff
@@ -229,18 +217,16 @@ fn wrapped_query(
 	query: String,
 	key: Value,
 ) -> Result<QueryResult> {
-	if publisher == *MITRE_PUBLISHER {
+	if publisher == *MITRE_PUBLISHER && MITRE_LEGACY_PLUGINS.contains(&plugin.as_str()) {
 		if query != *DEFAULT_QUERY {
 			return Err(hc_error!("legacy analyses only have a default query"));
 		}
 		match plugin.as_str() {
-			ACTIVITY_PHASE => db.activity_analysis(),
 			AFFILIATION_PHASE => db.affiliation_analysis(),
 			BINARY_PHASE => db.binary_analysis(),
 			CHURN_PHASE => db.churn_analysis(),
 			ENTROPY_PHASE => db.entropy_analysis(),
 			IDENTITY_PHASE => db.identity_analysis(),
-			FUZZ_PHASE => db.fuzz_analysis(),
```

Comment:
I would actually restore these two deleted lines from the match statement. the above `&& MITRE ...` does the check.